README.md

```diff
@@ -11,7 +11,21 @@
 
 ## Run with Docker
 
-Install Docker for your system: [https://www.docker.com/](https://www.docker.com/)
```

Comment:
We can remove the docker install guide in the  [New Starter Guide](https://confluence.ons.gov.uk/display/SDC/eQ%3A+New+Starter+Guide)